tox.ini

```diff
@@ -19,8 +19,10 @@ commands =
     py.test {posargs}
 
 [testenv:docs]
+deps = -r{toxinidir}/requirements/lint.txt
 changedir = {toxinidir}/docs
 commands =
+    rstcheck `find **/*.rst | xargs`
```

Comment:
I'm not sure if this is the right place, but looks _logical_ to me, what I don't like is to have to install all the deps from lint.txt. Maybe adding a docs.txt?

Reply:
Also the `--recursive` option from rstcheck was added reciently https://github.com/myint/rstcheck/commit/6102bec9a0096e7a855884a8019a0c1fa9f2b8a8, so when that version gets released I will update the command :)

Reply:
It's already released, `3.2`.

I also noted that you made a PR there! Hehe! Nice job! :)